client/src/main/kotlin/no/nav/su/se/bakover/client/pdf/SkattegrunnlagsPdfInnhold.kt

```diff
@@ -47,12 +49,34 @@ data class SkattegrunnlagsPdf private constructor(
                 skatt.søkers.årsgrunnlag.tilPdfJson(),
             ),
             eps = skatt.eps?.let { SkattPdfDataJson(it.fnr, hentNavn(it.fnr), it.årsgrunnlag.tilPdfJson()) },
+            begrunnelse = null,
+        )
+
+        fun Skattegrunnlag.lagPdfInnhold(
+            begrunnelse: String?,
+            navn: Person.Navn,
+            clock: Clock,
+        ): SkattegrunnlagsPdfInnhold = SkattegrunnlagsPdfInnhold(
+            saksnummer = null,
```

Comment:
Regner med det er noen planer for å fylle ut denne?

Reply:
Yep, kommer i andre PR